### PR TITLE
Added onboarding page for new Channels users

### PR DESCRIPTION
**Pull request by cwarnermm**
Added a new Channels onboarding page for users new to Mattermost Channels that parallels the new in-product onboarding flow.

### CODE REVIEW

**Comment by cwarnermm** [comment]
@esethna & @laneycs - To preview the new Channels onboarding page, visit http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/5391/messaging/get-started-mattermost-channels.html

Please note that there are currently placeholder links on this new page to other Channels topics. These placeholders will be resolved soon.

File: source/messaging/get-started-mattermost-channels.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+.. |lock| image:: ../images/lock-outline_F0341.svg
+  :height: 24px
+  :width: 24px
+  :alt: Private channels are identified with a Lock icon.
+
+To see all available public channels you can join, select the **Plus** |plus| icon at the top of the channel sidebar, then select **Browse Channels**. Search for channels by name or scroll through the list. Select **Join** next to any channel to become a member of that channel.
+
+.. tip:: 
+    Select **Find Channel** in the channel sidebar to see all of the channels you're currently a member of across all of your teams, including public and private channels, as well as all direct and group messages.
```
Comment (esethna) [inline source/messaging/get-started-mattermost-channels.rst:57]: The find channels doesn't actually display all your channels. It lists your recently visited channels and helps you jump to channels quickly by typing their name

**Review by esethna — APPROVED** [review]

File: source/welcome/get-started-mattermost-channels.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Channels and direct messages
+----------------------------
+
+Everyone who joins a Mattermost workspace is automatically added to two default channels: **Town Square** and **Off-Topic**. See our `Channel Types <https://docs.mattermost.com/channels/channel-types.html>`__ documentation for details.
+
+- **Town Square** is great for your team-wide communications, such as announcements.
+- **Off-Topic** is great for fun, social, non-work-related conversations. 
+
+Direct messages are private conversations between two people.
```
Comment (justinegeffen) [inline source/welcome/get-started-mattermost-channels.rst:23]: Are these three meant to be in a single sentence or in a list?

**Review by justinegeffen — APPROVED** [review]
This is awesome!! Well done! 

Just one question but approving for expediency. :)

File: source/welcome/get-started-mattermost-channels.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
+Get Started with Mattermost Channels
```
Comment (amyblais) [inline source/welcome/get-started-mattermost-channels.rst:1]: Does the page need to be added to the TOC?

**Pull request merged**